Examples/RTTI/RTTI_component/Examples/Cpp/RTTI_example.cpp

```diff
@@ -99,6 +99,48 @@ int main()
 		
 		animal = iter->GetNextAnimal();
 		assert(animal == nullptr);
+
+		// Test GetNextOptionalAnimal - class out parameter with bool return
+		std::cout << std::endl;
+		std::cout << "Testing GetNextOptinalAnimal:" << std::endl;
```

Comment:
```suggestion
		std::cout << "Testing GetNextOptionalAnimal:" << std::endl;
```
Check the spelling of Optinal -> Optional throughout.